torch/_inductor/config.py

```diff
@@ -12,6 +12,9 @@
 # Whether to enable printing the source code for each future
 verbose_progress = False
 
+# use fx aot graph cache
+fx_graph_cache = False
```

Comment:
Might be convenient to make this a number so you can easily invalidate the cache from user code